src/aat/resources/features/F-050/S-300.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "must return 403 when request provides authentic credentials without authorised access to the operation",
+  "_guid_": "S-300",
+  "_extends_": "F-050_Test_Data_Base",
+
+  "specs": [
+    "a detailed profile in CCD",
+    "uses an invalid authorization",
+    "has the 403 return code"
+  ],
+  "request": {
+    "headers": {
+      "Authorization": "InvalidAuthorization"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_403_Response",
+    "headers": {
+      "Transfer-Encoding" : "chunked"
+    },
+    "body": {
+      "timestamp" : "[[ANY]]",
+      "message" : "Access Denied",
+      "path" : "[[ANY]]"
```

Comment:
please remove the common fields that are now coming from base object.

Reply:
Removed Timestamp & path.